lib/ansible/modules/cloud/azure/azure_rm_virtualnetworkgateway.py

```diff
@@ -32,67 +32,65 @@
         required: true
     state:
         description:
-            - Assert the state of the VPN Gateway. Use 'present' to create or update VPN gateway and
-              'absent' to delete VPN gateway.
+            - State of the VPN Gateway. Use C(present) to create or update VPN gateway and C(absent) to delete VPN gateway.
         default: present
         choices:
             - absent
             - present
         required: false
     location:
         description:
-            - Valid azure location. Defaults to location of the resource group.
+            - Valid Azure location. Defaults to location of the resource group.
         required: false
     virtual_network:
         description:
-            - An existing virtual network with which the VPN Gateway will be associated. Required
-              when creating a VPN Gateway.
+            - An existing virtual network with which the VPN Gateway will be associated. Required when creating a VPN Gateway.
             - It can be the virtual network's name.
             - Make sure your virtual network is in the same resource group as VPN gateway when you give only the name.
-            - It can be the virtual network's resource id.
-            - It can be a dict which contains C(name) and C(resource_group) of the virtual network.
+            - It can be the virtual network's resource ID.
+            - It can be a dict which contains I(name) and I(resource_group) of the virtual network.
         aliases:
             - virtual_network_name
         required: true
     ip_configurations:
         description:
-            - List of ip configurations
+            - List of IP configurations
         suboptions:
             name:
                 description:
-                    - Name of the ip configuration.
+                    - Name of the IP configuration.
                 required: true
             private_ip_allocation_method:
                 description:
-                    - private ip allocation method.
+                    - Private IP allocation method.
                 choices:
                     - dynamic
                     - static
                 default: dynamic
             public_ip_address_name:
                 description:
-                    - Name of the public ip address. None for disable ip address.
+                    - Name of the public IP address. None for disable IP address.
             subnet:
                 description:
                     - ID of the gateway subnet for VPN.
                 default: GatewaySubnet
     gateway_type:
         description:
-            - The type of this virtual network gateway
+            - The type of this virtual network gateway.
         default: vpn
         choices:
             - vpn
             - express_route
     vpn_type:
         description:
-            - The type of this virtual network gateway
+            - The type of this virtual network gateway.
```

Comment:
I'm sure this is different than the `gateway_type`:
```suggestion
            - The type of this virtual private network.
```